src/main/resources/com/linecorp/armeria/server/docs/assets/js/armeria.js

```diff
@@ -24,6 +24,85 @@ $(function () {
     render(URI(window.location.href));
   });
 
+  function parseParametersDocString(docString) {
+    var parameters = {};
+    if (docString != null) {
+      var pattern = /@param ([^ \.]*)[ \.]*(.*)/g;
```

Comment:
This regexp doesn't stop with dot. It stops on newline. BTW, I'll update it with multiline things.